### ISSUE TITLE
Entry query is not working properly

### ISSUE THREAD

**Issue by dario-rodriguez** [contentstack/contentstack-management-javascript#40]
When I try to perform an entry query with especial characters it do not return the proper results.

Example: 

```
stack
  .contentType(entry.content_type_uid)
  .entry()
  .query({ query: { title: "Audit & Security" } })
  .find()
```

This query instead of returning the entry with that title, it returns many others not related. I tried the same behaviour with the delivery SDK and it works. In my scenario using the delivery sdk doesn't works because I need to fetch some queries not published

**Comment by uttamukkoji** [comment]
Hey @dario-rodriguez , Thanks for raising this issue. We will check this on our side and let you know once we release new version with the fix.

**Comment by dario-rodriguez** [comment]
Hello, @uttamukkoji, I also added a PR with the fix: #41

**Comment by uttamukkoji** [comment]
Hey @dario-rodriguez ,

Thanks for PR we have approved the PR #41. Closing the issue as its resolved.
